File: javascript/1159.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split("\n"),initial = parseInt(lines.shift());

while(initial !== 0){
    var sum = 0;
    for(let i = initial; i <= initial+9; i++){
        if(i % 2 === 0){
            sum+=i;
        }
    }
    console.log(sum);
    initial = parseInt(lines.shift());
}

File: C/1159.c
#include <stdio.h>

int main() {
    
    int initial, sum, i;
    
    scanf("%d", &initial);
    
    while(initial != 0){
        sum = 0;
        
        for(i = initial; i <= initial + 9; i++){
            if(i % 2 == 0){
                sum += i;
            }
        }
        
        printf("%d\n", sum);
        
        scanf("%d", &initial);
    }
    
    return 0;
}